pacemaker.spec.in

```diff
@@ -403,14 +403,26 @@ rm -rf %{buildroot}
 %postun
 %systemd_postun_with_restart pacemaker.service
 
+%pre remote
+systemctl --quiet is-active pacemaker_remote
+if [ $? -eq 0 ] ; then
+    touch %{_localstatedir}/lib/rpm-state/%{name}.restart_pacemaker_remote
+    systemctl stop pacemaker_remote >/dev/null 2>&1
+else
+    rm -f %{_localstatedir}/lib/rpm-state/%{name}.restart_pacemaker_remote
+fi
+
 %post remote
 %systemd_post pacemaker_remote.service
 
 %preun remote
 %systemd_preun pacemaker_remote.service
 
-%postun remote
-%systemd_postun_with_restart pacemaker_remote.service
```

Comment:
Well %postun is coming from the old package...
So we might even consider leaving the %systemd_postun_with_restart here for a smoother
way back to as it looks the standard way some day.
As we are anyway stopping the service in the first script to just start it in the last all the
scriptlets in between should anyway not see the case where they have to restart anything
as it is not running for them anyway.